apps/meteor/client/views/admin/integrations/edit/HistoryContent.tsx

```diff
@@ -1,16 +1,16 @@
+import { IIntegrationHistory, Serialized } from '@rocket.chat/core-typings';
 import { Skeleton, Box, Accordion } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
-import React from 'react';
+import React, { ReactElement } from 'react';
 
-import { AsyncStatePhase } from '../../../../hooks/useAsyncState';
 import HistoryItem from './HistoryItem';
 
-function HistoryContent({ data, state, onChange, ...props }) {
+function HistoryContent({ data, isLoading }: { data: Serialized<IIntegrationHistory>[]; isLoading: boolean }): ReactElement {
```

Comment:
why not es6+ ?

Reply:
IMO wouldn't change much. Also, using `function` makes sure the component is named, so in theory this should make it easier to debug.